index.html

```diff
@@ -35,6 +35,236 @@ <h3>LET'S CONNECT</h3>
+          <div class="project-link">
+            <div class="project-view">
+              <h3>See Project</h3>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="works-1">
+        <div class="snapshot">
+          <img src="images/snapshoot-3.png" alt="snapshot">
+        </div>
+        
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project_title">Tonic</h2>
+            <ul>
+              <li class="client">CANOPY</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">Back End Dev</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">2015</li>
+            </ul>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <div class="tags">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+
+          <div class="project-link">
+            <div class="project-view">
+              <h3>See Project</h3>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="works-1">
+        <div class="snapshot">
+          <img src="images/snapshoot-4.png" alt="snapshot">
+        </div>
+        
+        <div class="left-block">
+          <div class="primary-text">
+            <h2 class="project_title">Multi-Post Stories</h2>
+            <ul>
+              <li class="client">CANOPY</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">Back End Dev</li>
+              <li class="dot"><img src="vector/Counter.png" alt="counter"></li>
+              <li class="role">2015</li>
+            </ul>
+          </div>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <div class="tags">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+
+          <div class="project-link">
+            <div class="project-view">
+              <h3>See Project</h3>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+    </section>
+
+    <section class="about">
+      <div class="top">
+        <h1>About Myself</h1>
+
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through 
+          some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+
+        <div class="social-media">
+          <h3>LET'S CONNECT</h3>
+
+          <div class="flex-social">
+            <div><img src="vector/twitter.png" alt="Twitter"></div>
+            <div><img src="vector/linkedin_icon.png" alt="Linkedin"></div>
```

Comment:
reference | your copy
------------ | ------------
![image](https://user-images.githubusercontent.com/67957533/129376794-e5f0e8d2-0efe-448c-ac11-e9ca0090f0af.png) | ![image](https://user-images.githubusercontent.com/67957533/129376666-19dbe629-bb66-4abf-84ee-593d252a9426.png) 

- [x] Please apply the appropriate spacing here ☝️ 
```suggestion
            <div><img src="vector/Linkedin_icon.png" alt="Linkedin"></div>
```
- [x] This line `img src="vector/linkedin_icon.png"` should be `img src="vector/Linkedin_icon.png"`....the difference is the `L` starting Linkedin not linkedin ...this has caused the Linkedin icon not to show across your page